src/parser/filters/parseDescriptor.js

```diff
@@ -58,9 +63,15 @@ function getModifiers(parsableDescriptor) {
 		});
 	}
 
-	if (modifiers.length === 0 || remainingChars.trim().length > 0) {
-		return null;
+	if (modifiers.length === 0) {
+		const errorMsg = `"${symbol}" does not seems to be a chord`;
+		throw new NoSymbolFoundError(errorMsg, chord);
+	}
+	if (remainingChars.trim().length > 0) {
+		const errorMsg = `The chord descriptor "${descriptor}" contains unknown or duplicated modifiers: "${remainingChars}"`;
```

Comment:
remainingChars can sometimes be a bit weird:
- it might contain spaces
- it might contain lowercase version of the original modifier
- ...